amxmodx/amxmodx.cpp

```diff
@@ -1381,14 +1396,16 @@ static cell AMX_NATIVE_CALL show_menu(AMX *amx, cell *params) /* 3 param */
 	return 1;
 }
 
-static cell AMX_NATIVE_CALL register_plugin(AMX *amx, cell *params) /* 3 param */
+static cell AMX_NATIVE_CALL register_plugin(AMX *amx, cell *params) /* 5 param */
 {
 	CPluginMngr::CPlugin* a = g_plugins.findPluginFast(amx);
```

Comment:
```
enum { arg_title, arg_version, arg_author, arg_url, arg_description }
```

Reply:
it should be count from 1 or add first arg_count